pyEpiabm/pyEpiabm/property/household_foi.py

```diff
@@ -77,6 +77,8 @@ def household_foi(infector, infectee, time: float):
 
         """
         seasonality = 1.0  # Not yet implemented
+        isolation = infector.microcell.cell.isolation_house_effectiveness \
+            if infector.isolation_start_time is not None else 1
         false_pos = 1 / (1 - pyEpiabm.core.Parameters.instance().
                          false_positive_rate)
         infectiousness = (HouseholdInfection.household_inf(infector, time)
```

Comment:
I think it would be clearer to scale the infectiousness by isolation_house_effectiveness only. Although this will have the same outcome I think it makes more sense to scale infectiousness specifically if a person is isolated. This will also mean the structure of the foi is kept consistent across each type, and the interventions scaling will follow that for seasonality, false_pos, and care homes. 
I am also slightly unsure as to why isolation_house_effectiveness (and isolation_effectiveness later) are properties of the cell as opposed to global parameters? 
```suggestion
isolation_params =  pyEpiabm.core.Parameters.instance().intervention_params['case_isolation']
isolation_scaling = isolation_params['isolation_house_effectiveness']

infectiousness = (HouseholdInfection.household_inf(infector, time)
                              * seasonality * false_pos
                              * pyEpiabm.core.Parameters.instance().
                              household_transmission)
                              * (isolation_scaling if infector.is_isolated else 1)
```
Giving the person the property is_isolated in addition to isolation_start_time might also be useful to allow isolation to be triggered by case, icu etc thresholds as well as time.

Reply:
Could someone check if we need to make the isolation_house_effectiveness and isolation_effectiveness global parameters?

Reply:
We now read in these parameters in from the Parameters instead of making it a property of the cell.